bigbluebutton-html5/imports/ui/components/whiteboard/annotations/helpers.js

```diff
@@ -1,51 +1,32 @@
-// applies zooming to the stroke thickness
-const zoomStroke = (thickness, widthRatio, heightRatio) => {
-  let ratio;
-
-  ratio = (widthRatio + heightRatio) / 2;
-  return thickness * 100 / ratio;
-};
-
-const formatColor = (color) => {
-  // let color = this.props.annotation.annotation.annotation.color;
-  if (!color) {
-    color = '0'; // default value
-  }
-
-  if (!color.toString().match(/\#.*/)) {
-    color = colourToHex(color);
-  }
-
-  return color;
-};
-
 const colourToHex = (value) => {
   let hex;
-  hex = parseInt(value).toString(16);
+  hex = parseInt(value, 10).toString(16);
   while (hex.length < 6) {
     hex = `0${hex}`;
   }
 
   return `#${hex}`;
 };
 
-const getStrokeWidth = ((thickness, widthRatio, heightRatio) =>
-  zoomStroke(formatThickness(thickness), widthRatio, heightRatio)
-);
+const getFormattedColor = (color) => {
```

Comment:
Is this not the same as ` ... = color || '0'` ?

Reply:
`const getFormattedColor = (color = '0') => {...`

Reply:
@oswaldoacauan I am not sure what case it was supposed to protect from.
Just tested - doing it your way doesn't save from the case when color is `null`, it only saves us from when color value is `undefined` or when the variable is not passed to the function.
I guess it's better to go with `let _color = color || '0';`